tests/simulation/shard_rsm.cpp

```diff
@@ -769,6 +836,9 @@ void TestScanAllOneGo(ShardClient &client) {
   auto [result_size_2, next_id_2] = AttemptToScanAllWithExpression(client, v_id, 5, unique_prop_val_2);
   MG_ASSERT(result_size_2 == 1);
 
+  // AttemptToScanAllWithOrderByOnPrimaryProperty(client, v_id, 5);
```

Comment:
Why is this commented out?

Reply:
No idea, probably happened in one of the merge conflicts. I removed it.

Reply:
```suggestion
  AttemptToScanAllWithOrderByOnPrimaryProperty(client, v_id, 5);
```